deployment/base/resources-db.yaml

```diff
@@ -108,7 +108,6 @@ data:
     CREATE TABLE "Surgical_Plans" (
```

Comment:
don't we agree to delete this line too? 
https://github.com/BoredOtter/hms/blob/master/deployment/base/resources-db.yaml#L105